src/components/biggive-footer/biggive-footer.tsx

```diff
@@ -18,55 +32,206 @@ export class BiggiveFooter {
+              <HeadingTag class="heading" id="footer-tertiary-heading">
+                <slot name="nav-tertiary-title"></slot>
+              </HeadingTag>
+            </nav>
+
+            <div class="button-wrap">
+              <biggive-button colour-scheme="white" url="https://blog.thebiggive.org.uk/charities" label="For charities"></biggive-button>
+              <biggive-button colour-scheme="white" url="https://blog.thebiggive.org.uk/funders" label="For funders"></biggive-button>
+            </div>
           </div>
-        </div>
 
-        <div class="row row-bottom">
-          <div class="postscript-wrap">
-            <img class="fr-logo" src={getAssetPath('../assets/images/fundraising-regulator.png')} alt="Fundraising Regulator" />
+          <div class="row row-bottom">
+            <div class="postscript-wrap">
+              <img class="fr-logo" src={getAssetPath('../assets/images/fundraising-regulator.png')} alt="Fundraising Regulator" />
+
+              <nav class="nav nav-postscript" aria-label="Legal"></nav>
+            </div>
 
-            <nav class="nav nav-postscript" aria-label="Legal"></nav>
+            <div class="social-icon-wrap">
+              <slot name="social-icons"></slot>
+            </div>
+            <p>&copy; 2007 – 2022 The Big Give Trust (1136547) | Company number 07273065 | Dragon Court, 27-29 Macklin Street, London WC2B 5LX, United Kingdom</p>
           </div>
+        </footer>
+      );
+    };
+
+    const presetFooter = () => {
+      return (
+        <footer class="footer">
+          <div class="row row-top">
+            <nav class="nav nav-primary" aria-labelledby="footer-primary-heading">
+              <HeadingTag class="heading" id="footer-primary-heading">
+                <div slot="nav-primary-title">Match Funding Opportunities</div>
+              </HeadingTag>
+              <ul slot="nav-primary">
+                <li>
+                  <a href={makeURL('Blog', this.blogUrlPrefix, 'christmas-challenge')} class="icon-christmas">
+                    Christmas Challenge
+                  </a>
+                </li>
+                <li>
+                  <a href={makeURL('Blog', this.blogUrlPrefix, 'green-match-fund')} class="icon-green">
+                    Green Match Fund
+                  </a>
+                </li>
+                <li>
+                  <a href={makeURL('Blog', this.blogUrlPrefix, 'champions-for-children')} class="icon-children">
+                    Champions for Children
+                  </a>
+                </li>
+                <li>
+                  <a href={makeURL('Blog', this.blogUrlPrefix, 'women-girls-match-fund')} class="icon-women-girls">
+                    Women and Girls Match Fund
+                  </a>
+                </li>
+                <li>
+                  <a href={makeURL('Blog', this.blogUrlPrefix, 'kind2mind/')} class="icon-mental-health">
+                    Kind²Mind
```

Comment:
just to note Kind²Mind is the new new name, and kind2mind is the new URL. We need to make sure the URL works in live wordpress before this is used in prod.